descriptors/ModuleDescriptor-template.json

```diff
@@ -82,7 +82,7 @@
           "methods": ["POST"],
           "pathPattern": "/fqm/migrate",
           "permissionsRequired": ["fqm.migrate.post"],
-          "modulePermissions": ["perms.users.get", "permissions.users.item.get"]
+          "modulePermissions": ["perms.users.get", "permissions.users.item.get", "configuration.entries.collection.get", "inventory-storage.modes-of-issuance.collection.get"]
```

Comment:
mod-configuration is needed for migration V4 (adding times to dates); MOIs will be for this one